Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM --platform=linux/amd64 ubuntu:20.04 as builder
+
+RUN apt-get update
+RUN DEBIAN_FRONTEND=noninteractive apt-get install -y build-essential
+
+ADD . /repo
```

Comment:
nitpick: I tried to build the image a couple of times and changing the dockerfile rebuilt all layers after the COPY. You may want to add a .dockerignore for Dockerfiles in there to speed things up.

Reply:
I've been using a .dockerignore during development but have not pushed it to minimize conflicts with the upstream repo, which could add a .dockerignore to the repo in the future (if one doesn't already exist). Should I push one anyway?